src/atoms/Layout/Col/index.js

```diff
@@ -15,12 +15,12 @@ function Col(props) {
     xxLargeCols,
     fullwidth,
     flex,
-    colStyle,
+    style,
```

Comment:
@patrickkim This forces the user to use a `Col` component explicitly if they way to give it `style` or `className`, since these are reserved words. We should update the doc.